hooks/pre-commit.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-./hooks/git-format-staged --formatter "swiftformat --config .swiftformat stdin" "StreamChatClient_v3/**/*.swift"
-./hooks/git-format-staged --formatter "swiftformat --config .swiftformat stdin" "TestResources_v3/**/*.swift"
+./hooks/git-format-staged --formatter 'swiftformat --config .swiftformat stdin' 'StreamChatClient_v3/*.swift'
+./hooks/git-format-staged --formatter 'swiftformat --config .swiftformat stdin' 'TestResources_v3/*.swift'
```

Comment:
@b-onc will it work for subfolders?

Reply:
Yes, tested and working. Please test yourself too, so we're sure it's working!